examples/config-cluster-member0.json

```diff
@@ -0,0 +1,30 @@
+{
+  "distSpecVersion": "1.1.0",
+  "storage": {
+    "rootDirectory": "/tmp/zot-mem0"
+  },
+  "http": {
+    "address": "127.0.0.1",
+    "port": "9000"
+  },
+  "log": {
+    "level": "info"
+  },
+  "cluster": {
```

Comment:
config model looks good
but how do you know http:// or https://? I think you can look above and if `tls:` section is enabled or not

Reply:
Why are there two different config files? I see the storage dir is different, but it shouldn't be right? The configuration for _all_ members should be identical

Reply:
The db is needed for two reasons - dedupe and search/ui. I guess you want to keep the latter. In which case, a workaround would be to append instance-index to the rootDir. For example, /rootdir/1/repos, /rootdir/2/repos ... etc based on the index of the node in the cluster member list.